app/integrations/immich.py

```
@@ -540,93 +464,75 @@ def _normalize_immich_asset(
+                    MetadataSearchDto(id=asset_uuid),
+                )
+                items = search_result.assets.items
                 if items:
-                    return items[0]
-
-        # Raise for other error codes
-        response.raise_for_status()
-
+                    return items[0].model_dump(by_alias=True)
+                return {}
     except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)